drivers/at25xxx/at25xxx.c

```diff
@@ -81,12 +81,12 @@ static inline int _wait_until_eeprom_ready(const at25xxx_t *dev)
     return tries == 0 ? -ETIMEDOUT : 0;
 }
 
-static ssize_t _write_page(const at25xxx_t *dev, uint32_t pos, const void *data, size_t len)
+int at25xxx_write_page(const at25xxx_t *dev, uint32_t page, uint32_t offset, const void *data, size_t len)
 {
     /* write no more than to the end of the current page to prevent wrap-around */
-    size_t remaining = PAGE_SIZE - (pos & (PAGE_SIZE - 1));
+    size_t remaining = PAGE_SIZE - offset;
```

Comment:
Here the same thing could happen as in `at24cxxx_write_page()`, if `offset` > `PAGE_SIZE`.